tests/test_targets_realistic_coverage.py

```
@@ -139,7 +173,80 @@ def test_tdee_calculations_realistic(self):
                     pass
 
         except ImportError:
-            pass
+            pytest.skip("core.targets module is unavailable in this environment")
+
+        # Test error cases for BMR calculations
+        try:
+            # Test Katch formula without body fat
+            calculate_bmr(30, 70, 175, "male", formula="katch")
+            assert False, "Should raise ValueError for missing body fat"
```

Ruff (0.13.3)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)